### PR TITLE
fix(tensorflow): casting error on kwargs

### PR DESCRIPTION
**Pull request by bojiang**
## Checklist:

- [ ] My code follows the bentoml code style, both `make format` and
  `make lint` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Comment by bojiang** [comment]
requires https://github.com/bentoml/BentoML/pull/2665 to pass the ci

File: bentoml/_internal/utils/tensorflow.py [hunk 1]
```diff
@@ -163,9 +165,66 @@ def normalize_spec(value: t.Any) -> "tf_ext.TypeSpec":
...
+
+    Args:
+        signature (:code:`list[tf.TensorSpec]`):
+            signature of the tensorflow function.
+        *args (:code:`t.Any`):
+            positional arguments of the Python function.
+        **kwargs (:code:`t.Any`):
+            keyword arguments of the Python function.
+    """
+    import inspect
```
Comment (larme) [inline bentoml/_internal/utils/tensorflow.py:184]: do we need to put inspect import here? It's a builtin module so we don't need to lazy import it?

Comment (bojiang) [reply bentoml/_internal/utils/tensorflow.py:184]: I think lazy importing works too because it's builtin

Comment (aarnphm) [reply bentoml/_internal/utils/tensorflow.py:184]: inspect is fast to import and no need to lazily import it.

Comment (bojiang) [reply bentoml/_internal/utils/tensorflow.py:184]: @aarnphm 
Then are there any cons of lazily importing?

Comment (bojiang) [reply bentoml/_internal/utils/tensorflow.py:184]: utils functions rarely have dependencies overlap by might be imported everywhere. Thus lazy import by default should be good for them.

Comment (aarnphm) [reply bentoml/_internal/utils/tensorflow.py:184]: just for readability. Usually it is not as needed to do unless the library is very heavy to import.

Comment (bojiang) [reply bentoml/_internal/utils/tensorflow.py:184]: What readability?

Comment (aarnphm) [reply bentoml/_internal/utils/tensorflow.py:184]: don't worry. It is preference only.

**Review by larme — APPROVED** [review]
LGTM

File: bentoml/_internal/utils/tensorflow.py (hunk 1 above)
Comment (sauyon) [reply bentoml/_internal/utils/tensorflow.py:184]: We should probably be more structured about lazy importing for sure.

**Pull request merged**